pkg/launch/launchable.go

```diff
@@ -73,6 +76,15 @@ type LaunchableStanza struct {
 	// can be used to query a configured artifact registry which will provide the artifact
 	// URL. Version may not be used in conjunction with Location
 	Version LaunchableVersion `yaml:"version,omitempty"`
+
+	// Image: the name of the container image to run. This only applies when the launchable
+	// type is "docker"
+	Image DockerImage `yaml:"image,omitempty"`
```

Comment:
Do you think "version" would be suitable for specifying an image? There are a lot of other systems that look at a launchable's "version" field for informative or display purposes. Those too will  have to be changed to query an "image" field.

Reply:
good point, I'll look into that. The word "version" does seem a little weird for specifying the image name as well as its version, though, so I might investigate changing all the places that use the version field to call something that each launchable type can implement